glean-core/metrics.yaml

```diff
@@ -244,3 +244,16 @@ glean.error:
       - all_pings
     no_lint:
       - COMMON_PREFIX
+
+  preinit_tasks_timeout:
+    type: boolean
+    description:
+      Set to true if the tasks that are queued prior to Glean initialization
+      time out.
+    bugs:
+      - https://bugzilla.mozilla.org/1575848
+    data_reviews:
+      - https://bugzilla.mozilla.org/show_bug.cgi?id=1575848#c5
+    notification_emails:
+      - telemetry-client-dev@mozilla.com
```

Comment:
Since you're touching this, any chance you could change it to the glean mailing list instead?

Reply:
Yes, looks like all of them are pointing at this address, want me to change everything?

Reply:
All addresses updated to `glean-team@mozilla.com`